lib/algo/DMD.cpp

```diff
@@ -490,6 +630,15 @@ DMD::predict(double t)
     Vector* d_predicted_state_real = d_predicted_state_real_1->minus(
                                          d_predicted_state_real_2);
 
+    if (d_state_offset)
+    {
+        *d_predicted_state_real += *d_state_offset;
+    }
+    if (d_derivative_offset)
+    {
+        *d_predicted_state_real += *(d_derivative_offset->mult(t));
```

Comment:
It looks like this `mult` allocates a vector. Can this be done without a vector allocation?

Reply:
It is replaced by `plusAx` now.